docs/pydoc/config/joiners_api.yml

```
@@ -1,7 +1,7 @@
 loaders:
   - type: haystack_pydoc_tools.loaders.CustomPythonLoader
     search_path: [../../../haystack/components/joiners]
-    modules: ["document_joiner", "branch", "answer_joiner", "string_joiner"]
+    modules: ["document_joiner", "branch", "answer_joiner", "string_joiner", "list_joiner"]
```

nit: I would keep these by alphabetical order 
```suggestion
    modules: ["answer_joiner", "branch", "document_joiner", "list_joiner", "string_joiner"]
```